entity-service/src/main/kotlin/com/egm/stellio/entity/repository/Neo4jRepository.kt

```diff
@@ -175,6 +175,48 @@ class Neo4jRepository(
         return Pair(queryStatistics.nodesDeleted, queryStatistics.relationshipsDeleted)
     }
 
+    @Transactional
+    fun deleteEntityProperty(entityId: String, propertyName: String): Pair<Int, Int> {

```

Comment:
We already have deletePropertyFromEntity and deleteRelationshipFromEntity functions that are used in the entityService.
But both does not delete the children (properties and relationships) of the attribute.

Reply:
Should this new `deleteEntityProperty` replace the existing `deletePropertyFromEntity` where it is used (and same for relationships)?

Reply:
deletePropertyFromEntity and deleteRelationshipFromEntity are used in the appendEntityAttributes service function.
More precisely when we are appending an existing attribute with (allowing override option), so we are removing the old one and attaching the new one to the entity.
I think we should replace them with the new deleteEntityProperty and deleteEntityRelationship, since we don't need the children attributes anymore if the attribute itself was deleted.

Reply:
Go for it then